### ISSUE TITLE
chests

### ISSUE THREAD

**Issue by mitm001** [SoftbearStudios/mk48#183]
I know you like the chest exp floating after a sinking and its a nice feature. Its interesting in that it will recharge weapons and restore health also.

The problem I have with it is in regards to subs. When a sub sinks a ship it rarely can benefit from the floating exp as its not possible to surface to do so in most circumstances. Subs get a little over 50% of the exp they deserve from a kill. 

It also has the negative affect when I sink a ship whether in a fleet or their independent, by healing those nearby I may have damaged and recharges their weapons where I as a sub most likely am left with no ammunition and low health after the attack. 

You can improve things a little by allowing the chests to rise from depth to the surface rather than placing them immediately on the surface as is currently done.

This gives the sub a chance to collect some exp, heal a little and recharge weapons. I would suggest a similar speed as the chests currently have when being pulled into a vessel for a starting point. 

It would also be nice if a sub could attract exp when just below the surface rather than having to be fully surfaced to do so. Think of it like periscope depth. Maybe even have another button to allow the sub to only surface to that depth to collect exp. This would still expose them visually as shallow depth does now.

A proper balance could be struck with some tweaking I am sure.

**Comment by finnbear** [comment]
> its not possible to surface to do so in most circumstances

I don't see why not. It might take some planning ahead, but subs can start surfacing right before they get the kill, gobble up the crates, and then dive.

> You can improve things a little by allowing the chests to rise from depth to the surface rather than placing them immediately on the surface as is currently done.

They **do** actually rise from the depth of the sunken ship/submarine, albeit at an unrealistically fast speed (depth to surface in only ~2-4 seconds)

> A proper balance could be struck with some tweaking I am sure.

Yeah :)

**Comment by mitm001** [comment]
>  but subs can start surfacing right before they get the kill,

Yes, but its not often you can do it if a g5 is close or during fleet attacks. It also makes no sense a fleet can almost always benefit from a member being sunk with increased exp, health and ammo when the winner of the attack has mostly only negative results.

I cant count the times they suck up the exp with a lower level ship, upgrade several levels and counter attack at full strength. 

Its also very, very dangerous for any sub to surface as it gives your position away immediately.

Since you say the do already surface from depth, which I only have noticed with subs and when in close proximity of the sinking, maybe the subs should have a stronger attraction and increase the range some while submerged?
